src/middlewared/middlewared/plugins/kubernetes_to_docker/migrate.py

```diff
@@ -182,7 +182,7 @@ def migrate(self, job, kubernetes_pool, options):
                         'snapshot': snapshot,
                         'dataset_dst': destination_ds,
                         'dataset_properties': {
-                            k: v for k, v in DATASET_DEFAULTS.items() if k not in ['casesensitivity']
+                            k: v for k, v in DATASET_DEFAULTS.to_dict().items() if k != 'casesensitivity'
```

Comment:
We probably need to expand this to include `normalization` as well

Reply:
Working on simplifying this a bit more. We're setting `encyprtion=off in another function instead of keeping everything in one place.